packages/frontend/core/src/components/affine/ai-onboarding/general.dialog.tsx

```diff
@@ -181,28 +194,74 @@ export const AIOnboardingGeneral = ({
           />
         </main>
 
-        <footer className={styles.footer}>
+        <section
+          className={styles.privacy}
+          aria-hidden={!isLast || !!aiSubscription}
+        >
+          <Trans
+            i18nKey="com.affine.ai-onboarding.general.privacy"
+            components={{
+              a: (
+                <a
+                  className={styles.privacyLink}
+                  href="https://ai.affine.pro"
+                />
+              ),
+            }}
+          />
+        </section>
+
+        <footer className={styles.footer} data-is-last={isLast}>
           {isLast ? (
-            <>
-              <Button onClick={closeAndDismiss}>
-                {t['com.affine.ai-onboarding.general.try-for-free']()}
+            aiSubscription ? (
+              <Button
+                className={styles.baseActionButton}
+                size="large"
+                onClick={closeAndDismiss}
+                type="primary"
+              >
+                {t['com.affine.ai-onboarding.general.get-started']()}
               </Button>
-              <Button onClick={goToPricingPlans} type="primary">
-                {t['com.affine.ai-onboarding.general.purchase']()}
-              </Button>
-            </>
+            ) : (
+              <>
+                <Button
+                  className={styles.baseActionButton}
+                  size="large"
+                  onClick={closeAndDismiss}
+                >
+                  {t['com.affine.ai-onboarding.general.try-for-free']()}
+                </Button>
+                <Button
+                  className={styles.baseActionButton}
+                  size="large"
+                  onClick={goToPricingPlans}
+                  type="primary"
+                >
+                  {t['com.affine.ai-onboarding.general.purchase']()}
+                </Button>
+              </>
+            )
           ) : (
             <>
               {isFirst ? (
-                <Button onClick={onClose} className={styles.skipButton}>
-                  {t['com.affine.ai-onboarding.general.skip']()}
-                </Button>
+                <div />
```

Comment:
null ?

Reply:
To make sure the `Next` button align right because `justify-content: space-between` is used here.

Reply:
Okay, I am still using `null` here, dealing with alignment in CSS